BondageClub/Scripts/Drawing.js

```diff
@@ -1177,36 +1177,25 @@ function DrawCircle(CenterX, CenterY, Radius, LineWidth, LineColor, FillColor, C
 	Canvas.stroke();
 }
 
-/**
- * Draws a progress bar
- * @param {number} X - Position of the bar on the X axis
- * @param {number} Y - Position of the bar on the Y axis
- * @param {number} W - Width of the bar
- * @param {number} H - Height of the bar
- * @param {number} Progress - Current progress to display on the bar
- * @returns {void} - Nothing
- */
-function DrawProgressBar(X, Y, W, H, Progress) {
-	DrawRect(X, Y, W, H, "white");
-	DrawRect(X + 2, Y + 2, Math.floor((W - 4) * Progress / 100), H - 4, "#66FF66");
-	DrawRect(Math.floor(X + 2 + (W - 4) * Progress / 100), Y + 2, Math.floor((W - 4) * (100 - Progress) / 100), H - 4, "red");
-}
-
 /**
  * Draws a progress bar with color
- * @param {number} X - Position of the bar on the X axis
- * @param {number} Y - Position of the bar on the Y axis
- * @param {number} W - Width of the bar
- * @param {number} H - Height of the bar
- * @param {number} Progress - Current progress to display on the bar
- * @param {string} ColorFG - Color of the first part of the bar
- * @param {string} ColorFG - Color of the bar background
+ * @param {number} x - Position of the bar on the X axis
+ * @param {number} y - Position of the bar on the Y axis
+ * @param {number} w - Width of the bar
+ * @param {number} h - Height of the bar
+ * @param {number} value - Current progress to display on the bar
+ * @param {string} foreground - Color of the first part of the bar
+ * @param {string} background - Color of the bar background
```

Comment:
Based on the function below the types should look like this:
```suggestion
 * @param {string|null} [foreground=null] - Color of the first part of the bar
 * @param {string|null} [background=null] - Color of the bar background
```